qiita_db/support_files/patches/26.sql

```diff
@@ -0,0 +1,15 @@
+-- Jun 11, 2015
+
+-- Updating FASTA-Sanger -> FASTA_Sanger, needed so we can put restrictions on
+-- what kind of files the user can select in the GUI
+
+UPDATE qiita.filetype SET type='FASTA_Sanger' WHERE type = 'FASTA-Sanger';
```

Comment:
In line 35 of qiita_pet/uimodules/prep_template_tab.py we define which filetypes have access to which raw data. However, to do this we need the names to match and FASTA-Sanger is not a valid identifier.